### ISSUE TITLE
cache

### ISSUE THREAD

**Issue by lex111** [web-standards-ru/dictionary#360]
кешировать или кЭшировать?

Мне кажется первый вариант, потому что это уже прочно вошло в язык, поправьте, если не так, пожалуйста. Иначе на протяжении всей жизни я очень сильно ошибался, когда сознательно использовал именно первый вариант.

**Comment by pepelsbey** [comment]
В Википедии вариант по умолчанию [кэш](https://ru.wikipedia.org/wiki/%D0%9A%D1%8D%D1%88), я тоже пишу чере «э», несмотря на склонность к «е» в таких случаях. Ничего научнее не скажу, но кажется, что кэш более распространён, можно сравнить выдачу в поисковиках или тренды.

**Comment by lex111** [comment]
Да, я тоже обратил на это внимание, и чаще используется написание через "э", но не кажется, что это устаревшее написание? Как было в случае в вебом, который ранее писался через "э" или фронтенд (а не фронтэнд). В общем, можно использовать написание через "е", нужно добавлять это в словарь, и какой из вариантом тогда добавлять? В новых книгах часто видел написание именно через "е". Что думаешь по этому поводу?

**Comment by pepelsbey** [comment]
Думаю фиксация таких вариаций лежил за пределами задач этого словаря. Тут, строго говоря, и кэшу особенно делать нечего ;) В языке всегда существуют вариации нормы, которые через какое-то время сами становятся нормой. Нам бы слова посложнее устаканить и ввести в язык кириллицей, вот это — задачка поважнее.

**Comment by lex111** [comment]
Да, я понимаю, но может расширить цель словаря? Чтобы его можно было использовать не только для фронтенда, но вообще при переводе статей/книг по веб-разработке?

**Comment by pepelsbey** [comment]
Просто это цель чуть сложнее и требует большей работы ) Словарь пока немного заморожен. Если интересно взяться активнее — могу дать больше прав и как-то обсудить направление.

**Comment by pepelsbey** [comment]
Позвал в коллабораторы. Это должно позволить открывать-закрывать ишьи, пушить и более смело себя здесь вести :) Если будут вопросы или идеи — можно созвониться.

**Comment by lex111** [comment]
Спасибо, это, конечно, хорошо, но в сущности что решает? Я же не могу, например, самопроизвольно, добавить слово cache как "кеширование" (с буквой "e"). Нужно мнение народа, как-то можно их привлечь, я готов вести/дополнять словарь, но нужны разные мнения, в этом вся суть. Вот я открыл не так давно несколько ишью, и ни по одному из них не было дан комментарий (предложение), хотя я открыл не просто так, это вопрос к активности сообщества, наверное. Нет, у меня есть своё мнение (как переводить те или иные термины), но как их принимать вместе с сообществом, или открыть по каждому PR и позвать всех туда, для обсуждения?

**Comment by pepelsbey** [comment]
В сущности, это добавляет уверенности :) Теперь есть одобрение от авторов словаря, мол, действуйте: поднимайте шум в Слаке, начинайте новые переводы и обсуждения, призывайте людей. Есть ресурс соцсетей и подкаста Веб-стандартов, чтобы привлекать к проекту внимание.

**Comment by lex111** [comment]
Ок, понял, начну шумиху в скором времени, нужно самому подготовиться.

**Comment by lex111** [comment]
А что если я подготовлю опрос (термин, варианты перевода + плюс свой, если не нравятся остальные), например через формы гугл, как считаете такой вариант возможным?
